### PR TITLE
Send accept language header

### PR DESCRIPTION
**Pull request by jaroslav-kubicek**
<url>LiveURL: https://kiwicomsmart-faq-bpupecjshh.now.sh
</url>
</url>

### CODE REVIEW

**Comment by jaroslav-kubicek** [comment]
Conflicts resolved, cc @kantord & @git-toni

File: src/relay/environment.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ require('isomorphic-fetch');
 const uri = 'https://graphql.kiwi.com';
 const cache = new QueryResponseCache({ size: 200, ttl: 30 * 60 * 1000 });
 
-const buildFetchQuery = (token: string = '') => {
+const buildFetchQuery = (token: string = '', locale = 'en_US') => {
```
Comment (git-toni) [inline src/relay/environment.js]: `locale: string = 'en_US'`

File: src/relay/QueryRenderer.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  <UserContext.Consumer>
+    {({ loginToken }: UserContextType) => (
+      <LanguageContext.Consumer>
+        {language => {
+          const locale = frontendLanguageToLocale[language];
+
+          return (
+            <OriginalQueryRenderer
+              {...props}
+              environment={createEnvironment(loginToken, locale)}
```
Comment (git-toni) [inline src/relay/QueryRenderer.js]: The imported function `createEnvironment` does not accept these two parameters as defined [here](https://github.com/kiwicom/smart-faq/blob/94d2fe5a7e9e517ace5049cb5c698f8dd8421348/src/relay/environment.js#L48). It should accept both token and locale

Comment (jaroslav-kubicek) [reply src/relay/QueryRenderer.js]: 🤦‍♂️

Comment (jaroslav-kubicek) [reply src/relay/QueryRenderer.js]: fixed

**Pull request merged**